erpnext/crm/doctype/lead/lead.py

```diff
@@ -151,7 +151,7 @@ def make_quotation(source_name, target_doc=None):
 			"doctype": "Quotation",
 			"field_map": {
 				"name": "lead",
-				"lead_name": "customer_name",
+				"lead_name": "company_name",
```

Comment:
In quotation, there is no field called "company_name".

The correct fix will be, remove the line `"lead_name": "customer_name",`. `set_missing_values` does the mapping correctly.